src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,25 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.List;
+
+
+/**
+ * Sorts all persons in the address book to the user.
+ */
+public class SortCommand extends Command {
+
+    public static final String  COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE= COMMAND_WORD
+            + ": Sorts all persons in the address book as a list with index numbers.\n"
+            + "Example: " + COMMAND_WORD;
+
```

Comment:
Double empty lines can be removed, same for line 6

Reply:
Changes made to remove double empty lines